website/source/docs/platform/k8s/connect.html.md

```diff
@@ -174,10 +174,30 @@ Annotations can be used to configure the injection behavior.
 
 * `consul.hashicorp.com/connect-service-upstreams` - The list of upstream
   services that this pod needs to connect to via Connect along with a static
-  local port to listen for those connections. Example: `db:1234,auth:6789`
-  will start two local listeners for `db` on port 1234 and `auth` on port
-  6789, respectively. The name of the service is the name of the service
-  registered with Consul. This value defaults to no upstreams.
+  local port to listen for those connections. 
+
+    * Services
+   
+        The name of the service is the name of the service registered with Consul. You can optionally specify datacenters with this annotation. 
+        ```annotations:
```

Comment:
I think we still need an extra line between this code block and the preceding paragraph. The other fixes look good!